ts/components/ui/IOScrollView.tsx

```diff
@@ -80,6 +82,7 @@ type IOScrollView = WithTestID<
     excludeEndContentMargin?: boolean;
     /* Include page margins */
     includeContentMargins?: boolean;
+    refreshControl?: RefreshControlProps;
```

Comment:
```suggestion
    refreshControlProps?: RefreshControlProps;
```
What do you think?

Reply:
I agree with you! The proposed rename fits better the purpose of this property  🚀 a8af7541797ba9e5524d49a669baf7f868a861a8